CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 * [FEATURE] Ruler: Minimize chances of missed rule group evaluations that can occur due to OOM kills, bad underlying nodes, or due to an unhealthy ruler that appears in the ring as healthy. This feature is enabled via `-ruler.enable-ha-evaluation` flag. #6129
 * [FEATURE] Store Gateway: Add an in-memory chunk cache. #6245
 * [FEATURE] Chunk Cache: Support multi level cache and add metrics. #6249
+* [ENHANCEMENT] OTLP: Change otlp handler to consist with the Prometheus otlp handler. #6272
```

Comment:
Let's do multiple changelog entries for this PR. I can see 3 changes:
- By default we don't convert all attributes to labels. This is a change
- Target info will be behind a FF. Enabled by default.
- New config to specify `promote_resource_attributes` per tenant